src/test/java/unit/JsonParserTest.java

```diff
@@ -0,0 +1,66 @@
+package unit;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import parser.JsonParser;
+import parser.NoSuchFileException;
+import shop.Cart;
+import shop.RealItem;
+import shop.VirtualItem;
+
+import java.io.File;
+
+public class JsonParserTest {
+    private static Cart expectedCart;
+    private JsonParser parser = new JsonParser();
+
+    @BeforeAll
+    static void setUp() {
+        //Create new cart
+        expectedCart = new Cart("TestCart");
+
+        RealItem car = new RealItem();
+        car.setName("Audi");
+        car.setPrice(32026.9);
+        car.setWeight(1560);
+
+        VirtualItem disk = new VirtualItem();
+        disk.setName("Windows");
+        disk.setPrice(11);
+        disk.setSizeOnDisk(20000);
+
+        expectedCart.addRealItem(car);
+        expectedCart.addVirtualItem(disk);
+    }
+
+    @Test
+    void testJsonParser() {
+        //Write cart to the json file
+        parser.writeToFile(expectedCart);
+
+        //Newly created JSON file
+        File json = new File("src/main/resources/" + expectedCart.getCartName() + ".json");
```

Comment:
I suggest to cearte a constant for "src/main/resources/" because you are using this path in multiple tests.